### ISSUE TITLE
Viscera Cleanup Detail (246900) working.

### ISSUE THREAD

**Issue by lucifertdark** [ValveSoftware/Proton#1608]
# Compatibility Report
- Name of the game with compatibility issues: Viscera Cleanup Detail
- Steam AppID of the game: 246900

https://gist.github.com/lucifertdark/584c14275612e8e3c4a8070d428d8a02

I've managed to get this game working with Steam Play, here's how to do it......

1. Install the game.
2. find & remove the redist folder & anything that has dotnet in it's name, if you forget this delete the pfx folder & start again from here.
3. Use Protontricks to install dotnet452 (takes about 5 minutes). at the end click restart now button. "protontricks 246900 dotnet452"
4. download the 64bit vcruntime2012.exe from here https://www.microsoft.com/en-us/download/details.aspx?id=30679
5. use protontricks run taskmngr & use that to run the exe you downloaded. it may seem like nothing happens, just wait a few seconds then shut taskmngr & winetricks.
6. run the game but choose 64bit & only 64bit.
7. enjoy the game.
 ps. This also works for both the extras, Santa's Rampage & Shadow Warrior.

**Comment by HonkingGoose** [comment]
Thanks for your report. There is a report filed already for this game. Usually its preferred to have only 1 report per unofficially supported game.

#672

**Comment by lucifertdark** [comment]
I did search but it never showed up.

**Comment by kisak-valve** [comment]
This was transferred to https://github.com/ValveSoftware/Proton/issues/672#issuecomment-424694246.

**Comment by HonkingGoose** [comment]
@lucifertdark Hmm strange that you couldn't find the game, searching with the game-name is also unreliable for me though on Github.

I've found that searching by Steam appid number (in your report it would be 246900) is the best way to search. All reports filed here should have that number. This will usually work just fine.

But still: *thanks for reporting*, and don't be discouraged, I've made some mistakes on here too, that's all part of the learning process.
